docker/Dockerfile-test-scripts

```diff
@@ -25,3 +25,4 @@ RUN sh install_tinytex.sh fancyhdr
 
 # Re-installation of TinyTeX
 COPY docker/reinstall_tinytex.sh .
+RUN sh reinstall_tinytex.sh
```

Comment:
I surely did not elaborate enough on this aspect, but the idea is to provide a script that can be used to re-install TinyTeX and the previously-installed LaTeX packages in a (running) container in case the TeXLive repo becomes obsolete (errors like `Remote repository is newer than local (2018 < 2019)`). We probably want to add it to `test e.g. via` as follows
```bash
# reinstall TinyTeX and all existing packages in the container
docker run --rm install_scripts:test bash /reinstall_tinytex.sh
```

Reply:
ok, I see, agree then